projects/qgrid-core/mouse/mouse.state.d.ts

```diff
@@ -1,7 +1,13 @@
 import { Td } from '../dom/td';
 
 export declare type MouseStateCode = 'left' | 'right' | 'middle' | null;
-export declare type MouseStateStatus = 'release' | 'down' | 'up';
+export declare type MouseStateStatus =
+	| 'release'
```

Comment:
is it a valid syntax for the first item?

Reply:
this is an auto formating and yes seems its totally fine